README.md

```diff
@@ -11,9 +11,10 @@ A readable, easy and enjoyable way to manage portfolio databases using directori
 Pre-compiled binaries are available through [GitHub Releases](https://help.github.com/en/github/administering-a-repository/releasing-projects-on-github):
 
 ```shell
-$ wget https://github.com/ewen-lbh/portfoliodb/releases/latest/portfoliodb
+$ git clone https://github.com/ortfo/portfoliodb portfoliodb
```

Comment:
These instructions are not for compiling from source but for installing the pre-built binary.
Cloning the source is not useful, but downloading the latest binary is.